provider/provider_test.go

```diff
@@ -10,18 +10,18 @@ import (
 )
 
 var testProviders = map[string]terraform.ResourceProvider{}
```

Comment:
The standard naming for this is testAccProviders (at least this is what I'm seeing in other providers and doc). Could you rename it too?